src/main/java/org/owasp/wrongsecrets/challenges/ChallengesController.java

```diff
@@ -97,7 +97,7 @@ public String spoiler(@PathVariable("short-name") String shortName, Model model)
       Supplier<Spoiler> spoilerFromRandomChallenge =
           () -> {
             var challengeDefinition = findByShortName(shortName);
-            return challenges.getChallenge(challengeDefinition).getFirst().spoiler();
+            return challenges.getChallenge(challengeDefinition).get(0).spoiler();
```

Comment:
can you please undo the removal of `getFirst()`?

Reply:
ya i will and i am pushing the changed test scripts right now, extremely sorry for the delay

Reply:
now you can look into it